DefInjected/StatDef/Stats_Pawns_WorkMedical.xml

```diff
@@ -9,16 +9,16 @@
   <!-- EN: medical surgery success chance -->
   <MedicalSurgerySuccessChance.label>szansa na powodzenie operacji</MedicalSurgerySuccessChance.label>
   <!-- EN: The base chance that a character will succeed when performing a medical operation. The actual success chance is also affected by factors like facilities, room cleanliness, medicine used, the difficulty of the surgery, and inspirations. No matter how high the success chance, there is always a small chance of failure on any operation. -->
-  <MedicalSurgerySuccessChance.description>Prawdopodobieństwo, z jakim operacja się powiedzie, gdy postać przystąpi do jej przeprowadzenia. Porażka oznacza zmarnowanie czasu i zasobów, a nawet katastrofalne skutki dla pacjenta.</MedicalSurgerySuccessChance.description>
+  <MedicalSurgerySuccessChance.description>Podstawowa szansa na to, że ta postać odniesie sukces podczas wykonywania zabiegu operacyjnego. Powodzenie jest także zależne od czynników, takich jak wyposażenie, czystość pokoju, zastosowany lek, trudność operacji i inspiracji. Ale bez względu na to, jak wysoka jest szansa udanego przeprowadzenia zabiegu, zawsze istnieje ryzyko niepowodzenia przy każdej operacji.</MedicalSurgerySuccessChance.description>
```

Comment:
```suggestion
  <MedicalSurgerySuccessChance.description>Podstawowa szansa na to, że ta postać odniesie sukces podczas wykonywania zabiegu operacyjnego. Powodzenie jest także zależne od innych czynników, takich jak wyposażenie, czystość pokoju, zastosowany lek, trudność operacji i inspiracji. Bez względu na to, jak wysoka jest szansa na powodzenie operacji, zawsze istnieje ryzyko niepowodzenia.</MedicalSurgerySuccessChance.description>
```

Reply:
Zgadzam się z poprawką Jakuba.
Dodatkowo zniósłbym dwa przecinki: 
- Powodzenie jest także zależne od innych czynników, takich jak wyposażenie,
+ Powodzenie jest także zależne od innych czynników jak wyposażenie,

- Bez względu na to, jak wysoka jest szansa na powodzenie operacji,
+ Bez względu na to jak wysoka jest szansa na powodzenie operacji,

Reply:
Z tym   "jest szansa na **powodzenie** operacji, zawsze istnieje ryzyko **niepowodzenia**"  trochę dziwacznie to brzmi.